gufe/visualization/mapping_visualization.py

```diff
@@ -305,3 +307,57 @@ def draw_unhighlighted_molecule(mol, d2d=None):
         bond_colors=[{}],
         highlight_color=red,
     )
+
+
+def _concat_images(imgs, buffer_spacing=4, ncols=3)-> Image:
+    """
+        This helper function is constructing a Grid Image of given imgs.
```

Comment:
Even if it's private, this really could use with a better explanation of what the method does.